### ISSUE TITLE
How to apply torch.quantization.quantize_dynamic for conv2d layer?

### ISSUE THREAD

**Issue by Midhilesh29** [pytorch/tutorials#899]
I am working on quantizing resnet50 model. I tried to use the following command.

```
quantized_model = torch.quantization.quantize_dynamic(
    resnet18, {torch.nn.Conv2d,torch.nn.Linear}, dtype=torch.qint8
)
```

But only the linear layer has quaantized but not the convolutional layer. Can anyone help me how to dynamically quantize the convolutional layer?

**Comment by Willert98** [comment]
same problem,have some solution?

**Comment by kamata1729** [comment]
It seems dynamic quantization does not supports conv2d currently
https://discuss.pytorch.org/t/cannot-quantize-nn-conv2d-with-dynamic-quantization/66722/2

**Comment by CoinCheung** [comment]
Hi,

in the tutorials, the batchnorm layers are always fused into conv layers. Does this mean that quantization of batchnorm layers are not supported also, even with static quantization ?
